frontend/simple/views/NavBar.vue

```diff
@@ -107,15 +107,17 @@ export default {
         let response = await namespace.lookup(this.name)
         let identity = response.body
         console.log(`Retrieved identity ${identity}`)
-        this.$store.commit('login', this.name)
-        this.toggleModal()
+        this.$store.dispatch('login', this.name)
+        Vue.events.$once('login', () => {
```

Comment:
If it's cleaner to await on the dispatch function, then do that instead of using the events bus.